node/forceios.js

```diff
@@ -0,0 +1,314 @@
+        "remoteAccessConsumerKey": config.appid || "3MVG9Iu66FKeHhINkB1l7xt7kR8czFcCTUhgoA8Ol2Ltf1eYHOU4SqQRSEitYFDUpqRWcoQ2.dBv_a1Dyu5xa",
+        "oauthRedirectURI": config.callbackuri || "testsfdc:///mobilesdk/detect/oauth/done",
+        "oauthScopes": ["web", "api"],
+        "isLocal": config.apptype === 'hybrid_local',
+        "startPage": config.startpage || 'index.html',
+        "errorPage": "error.html",
+        "shouldAuthenticate": true,
+        "attemptOfflineLoad": false
+    };
+    // console.log("Bootconfig:" + JSON.stringify(bootconfig, null, 2));
+
+    fs.writeFileSync(path.join('www', 'bootconfig.json'), JSON.stringify(bootconfig, null, 2));
+    shelljs.exec('cordova prepare ios');
+    shelljs.popd();
+
+    // Inform the user of next steps.
+    var nextStepsOutput =
+        ['',
+         outputColors.green + 'Your application project is ready in ' + projectDir + '.',
+         '',
+         outputColors.cyan + 'To build the new application, do the following:' + outputColors.reset,
+         '   - cd ' + projectDir,
+         '   - cordova build',
+         '',
+         outputColors.cyan + 'To run the application, start an emulator or plug in your device and run:' + outputColors.reset,
+         '   - cordova run',
+         '',
+         outputColors.cyan + 'To use your new application in XCode, do the following:' + outputColors.reset,
+         '   - open ' + projectDir + '/platforms/ios/' + config.appname + '.xcodeproj in XCode',
+         '   - build and run',
+         ''].join('\n');
+    console.log(nextStepsOutput);
+    console.log(outputColors.cyan + 'Before you ship, make sure to plug your OAuth Client ID,\nCallback URI, and OAuth Scopes into '
+        + outputColors.magenta + 'www/bootconfig.json' + outputColors.reset);
+}
+
+//
+// Helper to create native application
+//
+function createNativeApp(config) {
+    var createAppExecutable = path.join(__dirname, '..', 'build', 'app_template_files', 'createApp.sh');
+
+    // Calling out to the shell, so re-quote the command line arguments.
+    var newCommandLineArgs = buildArgsFromArgMap(config);
+    var createAppProcess = exec(createAppExecutable + ' ' + newCommandLineArgs, function(error, stdout, stderr) {
+        if (stdout) console.log(stdout);
+        if (stderr) console.log(stderr);
+        if (error) {
+            console.log(outputColors.red + 'There was an error creating the app.' + outputColors.reset);
+            process.exit(3);
+        }
+    });
+}
+
+//
+// Helper for 'update' command
+//
+function updateApp(config) {
+    var appType = config.apptype;
+    if (appType !== 'native' && appType !== 'hybrid_remote' && appType !== 'hybrid_local') {
+        console.log(outputColors.red + 'Unrecognized app type: \'' + appType + '\'.' + outputColors.reset + 'App type must be native, hybrid_remote, or hybrid_local.');
+        usage();
+        process.exit(4);
+    }
+
+    //
+    // FIXME
+    // 
+
+    /*
+    // Copy dependencies
```

Comment:
I think so - we don't have it on the android side.
We could fix it to do pod update or cordova update but maybe it's best they use pod / cordova directly